controllers/lvmcluster_controller_test.go

```diff
@@ -67,6 +79,13 @@ var _ = Describe("LVMCluster controller", func() {
 			Eventually(func() bool {
 				err := k8sClient.Delete(ctx, lvmClusterOut)
```

Comment:
Change this to do the following:
Delete the lvmCluster. client.Delete should be successful
Eventually get(CSIDriver) should return errors.IsNotFound
Eventually get( LVMCluster) should return errors.IsNotFound